builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,182 @@
+
+To access the endpoint, a Moralis account and API key are required.
+
+Head over to the [signup page](https://admin.moralis.io/register) to create an account. Once you're in, navigate to the API keys section. There, you can generate your own unique API key and be sure to copy it for future use.
+
+![Moralis API key](/images/builders/integrations/indexers/moralis/moralis-1.webp)
+
+## Querying Moralis API {: #querying-moralis-api }
+
+With the API key, you can try Moralis' REST APIs. The following examples show how the EVM API works:
+
+Get the token balance of a wallet:  
+
+```bash
+curl --request GET \
+ --url 'https://deep-index.moralis.io/api/v2.2/0xd4d7fb1f98dD66f6D1f393E8e237AdF74c31F3ea/erc20?chain=moonbeam' \
+     --header 'accept: application/json' \
+ --header 'X-API-Key: INSERT_YOUR_API_KEY' 
+```
+
+Get the list of owners of an NFT:  
+
+```bash
+curl --request GET \
+ --url 'https://deep-index.moralis.io/api/v2.2/nft/0xfffffffffffffffffffffffffffffffffffffffff/owners?chain=moonbeam&format=decimal' \
+     --header 'accept: application/json' \
+ --header 'X-API-Key: INSERT_YOUR_API_KEY' 
+```
+
+For a comprehensive overview of the available APIs and their capabilities, please refer to Moralis' [official documentation](https://docs.moralis.io/web3-data-api/evm/reference).
+
+## Moralis SDK {: #moralis-SDK }
+
+Moralis has an SDK that allows developers to seamlessly integrate Moralis' API into their backend infrastructure. This SDK offers a wide array of features, including:
+
+- Data querying from both EVM and Solana APIs  
+- Integration of Web3 authentication  
+- A collection of utility functions for efficient data transformation and formatting  
+
+Moralis currently provides official SDK support for two primary programming languages:
+
+- NodeJS
+- Python
+
+To install Moralis SDK:
+
+=== "npm"
+
+    ```bash
+    npm install moralis
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add moralis
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm add moralis
+    ```
+
+=== "pip"
+
+    ```bash
+    pip install moralis
+    ```
+
+Once the SDK is ready, you can leverage it to query Moralis APIs. Here's an example of how to interact with the EVM API using a JSON RPC interface:
+
+=== "Javascript"
+
+    ```js
+    import Moralis from 'moralis';
+
+        try {
+          await Moralis.start({
+            apiKey: "INSERT_YOUR_API_KEY"
+    });
+
+          const response = await Moralis.EvmApi.block.getBlock({
+            "chain": "0x504",
+            "blockNumberOrHash": "1"
+    });
+
+          console.log(response.raw);
+    } catch (e) {
+          console.error(e);
+    }
+    ```
+
+=== "Python"
+
+    ```python
+    from moralis import evm_api
+
+    api_key = "INSERT_YOUR_API_KEY"
+
+    params = {
+          "chain": "moonbeam",
+          "block_number_or_hash": "1"
+    }
+
+    result = evm_api.block.get_block(
+          api_key=api_key,
+          params=params,
+    )
+
+        print(result)
+    ```
+
```

Comment:
I think we should add a sentence or two wrapping up the section and instructing users on where they can find more information on the SDK and the functions it exposes